### PR TITLE
Created steps in README to stand up local instance of the environment

### PR DESCRIPTION
**Pull request by BboyAkers**
Please provide feedback if there are any edits I can make!

### CODE REVIEW

**Comment by eggplantzzz** [comment]
Thanks for the addition @BboyAkers! I'll get this reviewed.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+cd trufflesuite.com/
+npm install
+```
+
+## Run dev build
+In terminal run: 
+```
+npm run dev
+```
+Navigate to **localhost:9000**/port specified in your terminal
```
Comment (gnidan) [inline README.md]: Can you please turn this into a hyperlink? It'll be easier that way!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+
+To view Browsersync settings navigate to **localhost:3001**
```
Comment (gnidan) [inline README.md]: (same with this)

File: README.md (hunk 1 above)
Comment (BboyAkers) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (BboyAkers) [reply README.md]: Done!

**Comment by eggplantzzz** [comment]
Thanks @BboyAkers !

**Comment by BboyAkers** [comment]
Haha much love to you all! Keep building out amazing stuff!

**Pull request merged**